src/nimdowpkg/windowmanager.nim

```diff
@@ -542,6 +542,18 @@ proc jumpToUrgentWindow(this: WindowManager) =
   urgentMonitor.setSelectedTags(tagID)
   this.display.warpTo(urgentClient)
 
+proc incWidthDiff(this: WindowManager) =
+  var layout = this.selectedMonitor.taggedClients.findFirstSelectedTag.layout
+  if layout of MasterStackLayout:
+    cast[MasterStackLayout](layout).widthDiff += 10
```

Comment:
Could you make the `widthDiff` value (10 here) adjustable via the config file?

If it's too much of a hassle, I could get to this in a few days

Reply:
How do I name that configuration?

Reply:
Maybe `resizeStep`?